src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/serviceprovider/performance/PerformanceReportProcessor.kt

```diff
@@ -28,7 +28,7 @@ class PerformanceReportProcessor(
       referralId = referral.id,
       contractReference = contract.contractReference,
       organisationId = contract.primeProvider.id,
-      currentAssigneeId = referral.currentAssignee?.id,
+      currentAssigneeEmail = referral.currentAssignee?.userName,
```

Comment:
should we use 

https://github.com/ministryofjustice/hmpps-interventions-service/blob/d599e6f0f6b892e3e5a237e5ddc208111f4190f1/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/HMPPSAuthService.kt#L99

here?